README.md

```diff
@@ -8,7 +8,8 @@ This is a monitoring plugin for [icinga](https://www.icinga.com) to check the st
 ## Usage
 Try the plugin at the command line like this:
 ```
-/usr/bin/php ./check_nextcloud.php -H cloud.example.com -u apiuser -p apipassword
```

Comment:
A mention of the old user/password should still be there for backwards compatibility. Maybe like so:

> Try the plugin at the command line like this:
> ```
> /usr/bin/php ./check_nextcloud.php -H cloud.example.com -T access-token
> ```
> 
> For backwards compatibility the old user/password method is also possible. Just remember that the user used has to be in the admin group, so use appropriate measures (a strong password, HTTPS only, etc.):
> ```
> /usr/bin/php ./check_nextcloud.php -H cloud.example.com -u apiuser -p apipassword
> ```